index.html

```diff
@@ -103,10 +103,731 @@
+				</div>
+	</div>
+	
+
+
+
+  
+	<div class="card-body"><span class="redunder fw-bolder text-dark"> From Manufacturing Hub to Innovation and R & D; Center </span>
+	<span class=" small font-roboto d-block text-muted fw-lighter">	With its pro-business agenda and open-border policy for investors, the Central European nation is hoping to bounce back from the COVID-19 crisis, while making greater strides toward innovation, research and development.</span>
+</div>
+</div>
+
+		<div class="card mt-3 font-roboto">
+				<form class="form-group card-header">
+			<label for="email" class="row p-1"> 
+			 <small class="col-2 logo-email"></small>
+			 <span class=" col-10 fw-bold p-1">  YOUR DAY WITH OUR TOP 5 ARTICLES </span> 
+			</label>
+				<input type="email" id="email" class="form-control" placeholder="Email address" required />
+			<a href="#" class="text-decoration-none"><div class="text-uppercase m-3 text-white fw-lighter mx-auto bgcolor text-center p-1">
+				Free Sign up <i class="fa fa-chevron-right"></i>
+			</div></a>
+		</form>
+	</div>
+
+	 <div class="card mt-3 ">
+		<div class=" position-relative ">
+			<img class="card-img-top" src="images/panama-online-photo.jpg" alt="Card image cap">
+			<div class="position-absolute bottom-0 Sp-insight p-1 start-0 w-15">
+				 SPONSORED INSIGHT
+					</div>
+		</div>
+		<div class="card-body"><span class="redunder fw-bolder text-dark"> Panama - Central America's Leading Investment Destination </span>
+		<span class="small font-roboto  d-block text-muted fw-lighter"> Panama is expected to rebound strongly in 2021, with growth forecasted between 4% and 7%. </span>
+	</div>
+	</div>
+
+
+	<div class="card mt-3">
+		<div class=" position-relative ">
+			<img class="card-img-top" src="images/american-school-doha.jpg" alt="Card image cap">
+			<div class="position-absolute bottom-0 Sp-insight p-1 start-0 w-15">
+				 SPONSORED INSIGHT
+					</div>
+		</div>
+		<div class="card-body"><span class="redunder fw-bolder text-dark">Why are Private Schools the Best Option for Your Child’s Future?</span>
+		<span class="small font-roboto  d-block text-muted fw-lighter">	If academic results, average class size, teacher proficiency, pupil diversity and religious orthodoxy of your child’s school are things that are important to you, you should be looking at a private school. </span>
+	</div>
+	</div>
+
+
+	<div class="card mt-3">
+		<div class=" position-relative ">
+			<img class="card-img-top" src="images/americas-best-nursing-homes-2020.png" alt="Card image cap">
+			<div class="position-absolute bottom-0 Sp-insight p-1 start-0 w-15">
+				 RANKINGS
+					</div>
+		</div>
+		<div class="card-body"><span class="redunder fw-bolder text-dark">America's Best Nursing Homes 2021 </span>
+		<span class="small font-roboto d-block text-muted fw-lighter"> In partnership with Statista, we established a ranking providing information and insights to help you during this challenging time. </span>
+	</div>
+	</div>
+</div></div>
```

Comment:
- [ ] This section should be centered in the tablet view, check original page for reference

Reply:
Thanks for the suggestion